README.md

```diff
@@ -711,7 +711,137 @@ config.customActionDelegate = YourCustomActionDelegate()
+
+```swift
+// Add this in your button tap handler
+let inboxViewController = IterableInboxNavigationViewController()
+present(inboxViewController, animated: true)
+```
+
+
+#### Mobile Inbox Customization Options
+
+The SDK provides ways of customizing the inbox UI to match how you'd like to use it in the app. Depending on how much customization you need, you can go from as little work as typing a name for `cellNibName` (and ensuring the existence of the corresponding XIB) on an instance of `IterableInboxNavigationViewController`, to creating a new class that inherits `IterableInboxViewController` or `IterableInboxNavigationViewController`, depending on your specific UI stack, and overriding the existing methods.
+
+
+##### inboxMode
+
+`IterableInboxViewController` can display inbox messages in two ways. The first and default, is `popup` mode where the mobile inbox will be presented in a modal fashion. The second, and most likely a better option for iOS UI guidelines, is the `nav` mode where the inbox message will be pushed onto the navigation stack. We recommend using the `nav` mode if you believe your in-app message is showing fine in a navigation stack.
+
+##### cellNibName
+
+You can completely change the look of mobile inbox by creating your own `UITableViewCell` XIB file in Storyboard.  Set the custom class of this XIB to be `IterableInboxCell` and connect the outlets. You can then set the `cellNibName` property of `IterableInboxNavigationViewController` to the name of your newly created XIB file name.
+
+##### navTitle
+
+You can set the property `navTitle` in `IterableInboxNavigationViewController` to set the title of mobile inbox. We recommend setting it to a value like 'Inbox'.
+
+#### Mobile inbox events and the events lifecycle
+
+Inbox messages are in-app messages that are delivered to mobile inbox. So the events that apply to in-app messages also apply to inbox messages.
+
+##### In App Send
+This event is triggered when an in-app is sent to a user.
+
+Important event properties
+
+- `createdAt` - when the in-app was sent
+- `campaignId`- the in-app campaign id
+
+##### In App Delivery
+This event is triggered when an in-app is received by user on their device.
+
+Important event properties
+
+- `createdAt` - when the in-app was received by the client
+- `campaignId`- the in-app campaign id
+
+
+##### Inbox Message Impression
+This event is triggered when an inbox message is shown in mobile inbox. An impression is registered if the inbox message is displayed in the viewport of mobile inbox. If the inbox message is not visible then an impression is not counted.
+
+Important event properties
+
+- `createdAt` - when the in-app was sent
+- `campaignId`- the in-app campaign id
+- `impressionCount` - how many times the message was in the visible viewport
+- `totalDuration` - the total duration the inbox message was shown in mobile inbox
+
+##### In App Open
```

Comment:
or if a message is sent as in-app + inbox then it is also visible when the message is displayed